### PR TITLE
fixing some issues from NavPanel

### PR DESCRIPTION
**Pull request by tchasinga**
new

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -11,64 +11,59 @@ import { fetchEbike } from './redux/ebike/ebikeSlice';
...
-      { message && <Message message={message} /> }
-      { addReserveMessage && <Message message={addReserveMessage} />}
-      { addEbikeMessage && <Message message={addEbikeMessage} />}
-
       <Router>
+        <NavigationPanel />
+        {message && <Message message={message} />}
+        {reservationMessage && <Message message={reservationMessage} />}
+
         <div className="App">
```
Comment (khanh-devos) [inline src/App.jsx]: Hello Jack, Could you explain why you delete so many configuration having no connection to the Navigation-pane in the App.jsx ?

**Review by khanh-devos — CHANGES_REQUESTED** [review]
This is my review.

File: src/App.jsx (hunk 1 above)
Comment (khanh-devos) [reply src/App.jsx]: And also the navigation pane does not show at the Reservation Form when I click it.

**Review by souad988 — APPROVED** [review]
@tchasinga  @khanh-devos  I think we can approve these changes now

**Pull request merged**